src/sql/workbench/parts/notebook/browser/outputs/gridOutput.component.ts

```diff
@@ -73,6 +75,14 @@ export class GridOutputComponent extends AngularDisposable implements IMimeCompo
 		}
 	}
 
+	@Input() set hover(value: boolean) {
+		// only reaction on hover changes
+		if (this._hover !== value) {
+			this.toggleActionbar(value);
+			this._hover = value;
```

Comment:
I assume the actionbar stays visible even after moving the mouse over to it? Looking through it seems like it's added as a child of the div but good to verify

Reply:
Correct. It doesn't disappear when you mouse over it 😄